src/System.Net.Http/src/System/Net/Http/HttpClient.cs

```diff
@@ -394,7 +383,17 @@ public Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, HttpCompl
             }
             finally
             {
-                linkedCts.Dispose();
+                try
+                {
+                    // When a request completes, dispose the request content so the user doesn't have to. This also
+                    // helps ensure that a HttpContent object is only sent once using HttpClient (similar to HttpRequestMessages
+                    // that can also be sent only once).
+                    request.Content?.Dispose();
+                }
+                finally
+                {
+                    linkedCts.Dispose();
+                }
             }
```

Comment:
Is the extra try/finally needed here?  Is it expected that Dispose will throw?  If that's a danger, is it really bad for the CTS to *not* be disposed? 

(This is mostly just curiosity, as clearly things will work they way you've written it.)